src/System.Security.Cryptography.Csp/tests/RC2CryptoServiceProviderTests.cs

```diff
@@ -27,5 +31,46 @@ public static void RC2KeySize()
                 Assert.Throws<CryptographicException>(() => rc2.KeySize = 128 + 8);
             }
         }
+
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public static void UseSalt_Success()
+        {
+            using (var rc2 = new RC2CryptoServiceProvider())
+            {
+                Assert.False(rc2.UseSalt);
+                rc2.UseSalt = true;
+                Assert.True(rc2.UseSalt);
+            }
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        public static void UseSalt_Throws_Unix()
+        {
+            using (var rc2 = new RC2CryptoServiceProvider())
+            {
+                Assert.False(rc2.UseSalt);
+                Assert.Throws<PlatformNotSupportedException>(() => (rc2.UseSalt = true));
```

Comment:
rc2.UseSalt = rc2.UseSalt (or directly assign it to false); just to show that we let it self-assign.